src/test/java/uk/gov/companieshouse/efs/api/email/mapper/EmailMapperFactoryTest.java

```diff
@@ -57,30 +57,6 @@ void setUp() {
             .withPaymentReportEmailMapper(paymentReportEmailMapper).build();
     }
 
-    @Test
```

Comment:
Not sure why validation was removed from `build()`. This test for validation of `this.confirmationEmailMapper` is surely necessary.